README.md

```diff
@@ -45,7 +45,12 @@ $ coreos-assembler init https://github.com/coreos/fedora-coreos-config
 The specified git repository will be cloned into `/srv/coreos/src/config`.
 
 If you're doing something custom, you likely want to fork that upstream
-repository.
+repository. You can pass an absolute local path to work from an existing
+repository:
+
+```
+$ coreos-assembler init /path/to/my/fedora-coreos-config
```

Comment:
i don't think this is true. or rather it highly depends on the mount points you set up. I think I'd rather smooth out the [environment variable approach](https://github.com/coreos/coreos-assembler/issues/75#issuecomment-422930394) and add it here to the README

Reply:
Right, if you have `alias coreos-assembler='podman ...'` it might not work. Hmm.. and that's what advertised on the `README` right now. Fair enough, I'll close this for now.

Reply:
OK, updated just for the OSTree casing bit. :arrow_down: